File: WinForms/EqDemoWinFormsNet4VB/Models/NWind/Category.vb
Imports System
Imports System.ComponentModel.DataAnnotations
Imports System.ComponentModel.DataAnnotations.Schema
Public Class Category

    <DatabaseGenerated(DatabaseGeneratedOption.None)>
    <Column("CategoryID")>
    Public Property Id As Integer
    Public Property CategoryName As String
    Public Property Description As String

    <ScaffoldColumn(False)>
    Public Property Array As Byte()


End Class

File: Wpf/EqDemoWpfNet6/Models/NWind/Category.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace EqDemo.Models
{

    public class Category
    {

        [DatabaseGenerated(DatabaseGeneratedOption.None)]
        [Column("CategoryID")]
        public int Id { get; set; }

        public string CategoryName { get; set; }

        public string Description { get; set; }

        [ScaffoldColumn(false)]
        public byte[] Picture { get; set; }

    }
}